### PR TITLE
#2 Welcome page content repurposed to "What is PayID?"

### PR DESCRIPTION
**Pull request by LoisRP**
## High Level Overview of Change
Note: I meant this to be a draft pull request like https://github.com/payid-org/payid-documentation/pull/125/)

I love the visual look and content of the new home page for docs.payid.org, but that update lost the welcome page information about PayID. I think we should still have that, and here is one option to do so, but other options welcome.

Note: This branch is based on the branch that updated the "Get Started" content.

### Context of Change

Changed Welcome page to "What is PayID?" in case of confusion with the home page. But other suggestions are welcome. 

### Type of Change

- [x] Documentation Updates

## Before / After

"What is PayID?" page added that has the content of the old "Welcome to PayID" page. 

## Test Plan

Built and viewed locally.

## Future Tasks
Consider other options for including this content.

### CODE REVIEW

File: docs/getting-started-sandbox.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+id: getting-started-sandbox
+title: Getting Started With PayID Sandbox
+sidebar_label: Getting Started With PayID Sandbox
+---
+
+With the [PayID Sandbox](https://payid.org/sandbox), you can experiment with PayID without deploying your own server. Although the PayID Sandbox can be used to create PayIDs, do not use these PayIDs that rely on this virtual server to send or receive real cryptocurrency. The PayID Sandbox does not directly support Verifiable PayID.
```
Comment (ryangyoung) [inline docs/getting-started-sandbox.md:7]: > do not use these PayIDs that rely on this virtual server to send or receive real cryptocurrency.

This is probably worth a separate callout (a Caution, perhaps).

Comment (ryangyoung) [reply docs/getting-started-sandbox.md:7]: Also suggest small tweak:

> do not use PayIDs that rely on this virtual server to send or receive real cryptocurrency.

(remove "these")

File: docs/getting-started-sandbox.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+You can also experiment with PayID by using [PayID CLI](payid-cli), which does support Verifiable PayID.
+
+The sandbox allows you to experiment with PayID, but does not create an application that allows you to send or receive currency using the PayID.
+
+You are encouraged to copy the cURL or JavaScript code snippets that the sandbox provides and adapt them to your app.
+
+Requirements:
+
+- GitHub account. You will use this GitHub account to log in to PayID Sandbox.
+- Crypto addresses on other wallets or exchanges. You can map your PayID to these crypto addresses.
```
Comment (ryangyoung) [inline docs/getting-started-sandbox.md:18]: Not for this PR, but in the future would be great to see a link to another resource about wallets or how to generate the addresses.

File: docs/getting-started-sandbox.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## Open PayID Sandbox and Create a User
+
+1. Open [PayID Sandbox](https://payid.org/sandbox) and log in with your GitHub account.
+   The sandbox sets up a virtual server for you at _github_user_name_.sandbox.payid.org.
```
Comment (ryangyoung) [inline docs/getting-started-sandbox.md:23]: > _github_user_name_.sandbox.payid.org

Trivial but consider placing all of the above in code format and making the variable obnoxiously obvious, i.e.:

> `<YOUR_GITHUB_USERNAME>.sandbox.payid.org`

File: docs/getting-started-sandbox.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+## Get User
+
+**Get User** provides the same information as **Get Payment Information**, but **Get User** does so through the private Admin API.
+
+1. Click **Get User** on the left.
+2. Enter the PayID you want to look up.
+3. Select either **cURL** or **JavaScript**.
+4. Click **Run**. The JSON response provides information about the PayID.
+
+If you enter a nonexistent PayID, you get the response "Failed to find the PayID <_value_>".
```
Comment (ryangyoung) [inline docs/getting-started-sandbox.md:70]: > "Failed to find the PayID <_value_>"

Trivial again, but consider making this code font if it's part of the response. And if you capitalize the variable, you won't need to use the underscores, too. 

> `Failed to find the PayID <NONEXISTENT_PAYID_VALUE>`

**Review by ryangyoung — COMMENTED** [review]
This looks great @LoisRP. Left a few minor stylistic suggestions and some things to follow up on in future PRs.

**Comment by Stormtv** [comment]
run lint and ping me when you need this reviewed

File: docs/what-is-payid.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
-id: welcome-to-payid
-title: Welcome to PayID
-sidebar_label: Welcome to PayID
+id: what-is-payid
+title: What is PayID?
+sidebar_label: What is PayID?
```
[the remark this answers is not in the document]
Comment (LoisRP) [reply docs/what-is-payid.md:4]: Did a bit on this.

File: docs/getting-started-sandbox.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+   }
+   ```
+
+## Get Payment Information
+
+**Get Payment Information** is a public API, and does not require your Admin API token. You can use this API for any PayID.
+
+1. Click **Get Payment Information** on the left.
+2. Enter the PayID. On the sandbox, you are restricted to PayIDs on your server. Enter the PayID you just created, or another one you happen to know. Select either **cURL** or **JavaScript** at the top right.
+3. Click **Run**. The JSON response provides information about the PayID.
```
[the remark this answers is not in the document]
Comment (LoisRP) [reply docs/getting-started-sandbox.md:59]: That is the response. I've made that clearer.

[the remark this answers is not in the document]
Comment (LoisRP) [reply docs/getting-started-sandbox.md:59]: Now I see what you mean. Have added there as well.

File: docs/getting-started-sandbox.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+
+## Get User
+
+**Get User** provides the same information as **Get Payment Information**, but **Get User** does so through the private Admin API.
+
+1. Click **Get User** on the left.
+2. Enter the PayID you want to look up.
+3. Select either **cURL** or **JavaScript**.
+4. Click **Run**. The JSON response provides information about the PayID.
```
[the remark this answers is not in the document]
Comment (LoisRP) [reply docs/getting-started-sandbox.md:68]: Done.

File: docs/getting-started-sandbox.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+     ]
+   }
+   ```
+
+## Get Payment Information
+
+**Get Payment Information** is a public API, and does not require your Admin API token. You can use this API for any PayID.
+
+1. Click **Get Payment Information** on the left.
+2. Enter the PayID. On the sandbox, you are restricted to PayIDs on your server. Enter the PayID you just created, or another one you happen to know. Select either **cURL** or **JavaScript** at the top right.
```
[the remark this answers is not in the document]
Comment (LoisRP) [reply docs/getting-started-sandbox.md:58]: You can't change the server in the UI. The sandbox only allows your specific local server.

File: docs/getting-started-sandbox.md (hunk 1 above)
Comment (LoisRP) [reply docs/getting-started-sandbox.md:7]: Done

File: docs/getting-started-sandbox.md (hunk 2 above)
Comment (LoisRP) [reply docs/getting-started-sandbox.md:18]: On its way!

File: docs/getting-started-sandbox.md (hunk 3 above)
Comment (LoisRP) [reply docs/getting-started-sandbox.md:23]: Done.

File: docs/getting-started-sandbox.md (hunk 4 above)
Comment (LoisRP) [reply docs/getting-started-sandbox.md:70]: Added code font. Kept quotes, because they are part of the response.

**Review by Stormtv — APPROVED** [review]

**Pull request merged**